bin/cron/report_activejob_metrics

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+
+METRICS_NAMESPACE = 'CDO/ActiveJob'
```

Comment:
We are not currently prefixing our custom metric namespaces with `CDO/` but I'm proposing we start. Perhaps it's redundant. Unfortunately you cannot delete metrics, so the `CDO/ActiveJob` namespace will remain in Cloudwatch for at least 15 months, until the metrics I've already pushed expire.

Reply:
I support standardizing on a namespace for custom metrics~ I’d prefer something like `code-dot-org`

Reply:
From a usability perspective, I'm leaning acronym, but could go either way. The place where we would most often deal with the prefix is when searching for metrics. These would all have the prefix appended.
![image](https://github.com/code-dot-org/code-dot-org/assets/7014619/6afd232c-3911-43c4-95c2-97fb6ef49a91)

Reply:
Any standardized name is better than none, but also every new engineer starts looking at our codebase and needs about 5 weeks before they realize what `CDO` stands for 😂